packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -68,11 +69,13 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
   useCPURendering = false;
   use16BitTexture = false;
   private _FrameOfReferenceUID: string;
-  private inverted = false;
 
   // Viewport Properties
-  // TODO: similar to setVoi, this is only applicable to first volume
-  private VOILUTFunction: VOILUTFunctionType;
+  private initViewportProperties = new Map<string, VolumeViewportProperties>();
+  private currentViewportProperties = new Map<
+    string,
+    VolumeViewportProperties
+  >();
 
```

Comment:
I expected these sginitures be similar to stackViewport, I expected a global one and a perVolumeIdProperties.

What does init and current mean here? and why do we need to have this inconsistency between viewports?

Reply:
Yes you're right. I'll change it to fit the way we do it in the stack viewport